K2Bridge.Tests.End2End/ParallelApiTest.cs

```diff
@@ -163,10 +163,10 @@ public void CompareElasticKusto_WhenMSearchSort_ResponsesAreEquivalent(string qu
         }
 
         [Test]
-        [Description("/_search index list Kibana query")]
+        [Description("/_resolve/index Kibana query")]
         public void CompareElasticKusto_WhenSearch_ResponsesAreEquivalent()
         {
-            var es = ESClient().Search();
+            var es = ESClient().Search($"{INDEX}");
```

Comment:
so now you must pass an index?

Reply:
So that extra indices (like the kibana index) can be removed from the ES results.
I now realize that this index only exists if Kibana has been launched, which is not normally the case in end-to-end tests... But it was the case on my machine :) So maybe this is not required indeed.

Reply:
I have reverted this change; it works fine as long as Kibana is not launched before running the end-to-end tests.